src/edu/csus/ecs/pc2/core/imports/clics/CLICSAward.java

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * Single CLICS Award.
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+
+@JsonTypeInfo(use=Id.NONE)
+public class CLICSAward {
+
+    /**
+     * Identifier of the award.
+     */
+    @JsonProperty
+    private String id;
+
+    /**
+     * Award citation, e.g. "Gold medal winner".
+     */
+    @JsonProperty
+    private String citation;
+
+    /**
+     * JSON array of team ids receiving this award.
+     * 
+     * No meaning must be implied or inferred from the order of IDs. If the value is null this means that the award is not currently being updated. If the value is the empty array this means that the
+     * award is being updated, but no team has been awarded the award at this time.
+     */
+    @JsonProperty
+    private String [] team_ids;
+    
+    public CLICSAward() {
+        super();
+    }
+    /**
+     * Award with teams.
+     * @param id  identifier for award, ex winner
+     * @param citation ex. Contest winner
+     * @param team_ids lsit of ids
+     */
+    public CLICSAward(String id, String citation, List<String> team_ids) {
+        super();
+        this.id = id;
+        this.citation = citation;
+        this.team_ids = (String[]) team_ids.toArray(new String[team_ids.size()]);
+    }
+    
+    /**
+     * Award with team.
+     * @param id  identifier for award, ex winner
+     * @param citation ex. Contest winner
+     * @param teamid  single team id
+     */
+    public CLICSAward(String id, String citation, String teamid) {
+        super();
+        this.id = id;
+        this.citation = citation;
+        String [] teamIds = {teamid};
+        this.team_ids = teamIds;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getCitation() {
+        return citation;
+    }
+
+    public String[] getTeam_ids() {
```

Comment:
The return type of this method seems inconsistent with the structure of the constructors.  Specifically, as pointed out above, there is a constructor that accepts `List<String>` for team_ids, but no corresponding constructor accepting `String []`.  That actually makes sense; external clients (callers) shouldn't know the internal storage type of elements (in this case, the fact that team_ids are stored as an _array_ of Strings).  However, the return type of this method defeats that; it exposes the fact that team_ids are stored as arrays of Strings.  It seems like the return type of this method should be `List<String>`, for consistency with the constructors and to avoid exposing the internal (array) storage type.